lib/counters/inMemory.js

```diff
@@ -1,15 +1,17 @@
 var _ = require('lodash');
-var lruCache = require('lru-cache');
+var LRUCache = require('lru-cache');
 
 module.exports = function init(options) {
-  var size = _.get(options, 'size') || 1000;
-  var cache = lruCache({ max: size });
+  const size = _.get(options, 'size') || 1000;
+  const cache = new LRUCache({
+    max: size,
+  });
```

Comment:
I'd change these back to `vars`, although I agree `const` would be better. The rascal code base is still ES5, even though it relies on modules that use `const` so there's no problem changing these. I'd rather keep the code base consistent for now and update the whole codebase at some point.